errors

```diff
@@ -1,3 +1,12 @@
 exports.wrongPathError = (req, res) => {
-res.status(404).send({ msg: "path not found"})
-};
\ No newline at end of file
+  res.status(404).send({ msg: "path not found" });
+};
+
+exports.psqlErrors = (err, req, res, next) => {
+  if (err.code === "22P02") {
+    res.status(400).send({ msg: "invalid input" });
+  } else if (err) {
+    const { status, msg } = err;
```

Comment:
If there is a `status` key on the error object then this is not a psql error so it should be handled by a custom error handler and not the psql error handler. You can invoke next with the error to get from one error handler to the next should you need to.

Reply:
Thanks, I understand, I've created a custom error and altered the code so that the custom error will be picked up when there's a err.status.